Sources/Preferences/Localization.swift

```diff
@@ -15,13 +15,9 @@ struct Localization {
 			"de": "Einstellungen",
 			"el": "Προτιμήσεις",
 			"en": "Preferences",
-			"en-AU": "Preferences",
-			"en-GB": "Preferences",
```

Comment:
I compared all these strings using a script to make sure they're 100% the same. In case the translation for various variants of English changes, they can be put back in just by adding them to the dictionary, but for now, there's no use in keeping them.